tests/data/virtual-server-certmanager/self-signed.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
Are these copies of the other example objects? Can we store these in a single place and render them in docs and include from in tests from a single store?

Reply:
This particular file is, but the rest are a different. Having a single store is a great suggestion and is something that we implement, but probably outside the scope of this PR.